client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx

```diff
@@ -375,7 +374,10 @@ const TestPlanUpdaterModal = ({
                                         </p>
                                     );
                                 }
-                                if (runsWithResults.length === 0) {
+                                if (
+                                    !runsWithResults ||
+                                    runsWithResults.length === 0
+                                ) {
                                     return (
                                         <p>
                                             There are no test results associated
```

Comment:
I'd actually vote to `return ''`  instead of "There are no test results associated with this report." because when I was using the modal the message confused me. It sounded almost like an error message.